pkg/api/datasources.go

```diff
@@ -231,6 +238,27 @@ func AddDataSource(c *models.ReqContext, cmd models.AddDataSourceCommand) respon
 	})
 }
 
+func (hs *HTTPServer) StoreDataSourceInVCS(ctx context.Context, ds models.DataSource) error {
+	if gitops, ok := hs.Cfg.FeatureToggles["gitops"]; !ok || !gitops {
```

Comment:
```suggestion
	if gitops, ok := hs.Cfg.FeatureToggles["gitops"]; !ok || !gitops || hs.PluginManager.VersionedControlStorage() == nil {
```
When the feature toggle will be removed, we still want to ensure that there is a plugin before enabling this feature, right?

Reply:
Alright